ui/js/values.js

```diff
@@ -35,6 +37,7 @@ treeherder.value("thPlatformMap", {
     "osx-cross-add-on-devel": "OS X Cross Compiled addon",
     "osx-cross-devedition": "OS X Cross Compiled DevEdition",
     "macosx64-stylo": "OS X 10.10 Stylo",
+    "macosx64-stylo-disabled": "OS X 10.10 Stylo-Disabled",
```

Comment:
Hmm though some of the existing jobs use the old naming.

There seems to be a lot of inconsistency in these job definitions :-(
(Probably not helped by taskcluster vs buildbot namings)